src/holodeck/environments.py

```diff
@@ -212,6 +212,7 @@ def _load_scenario(self):
                 'location': [0, 0, 0],
                 'rotation': [0, 0, 0],
                 'agent_name': agent['agent_type'],
+                'max_height': sys.maxsize,
```

Comment:
Would 
`'max_height': 0` be better than `'max_height': sys.maxsize`? I guess it doesn't matter _too_ much, but just want to make sure.